### PR TITLE
Improved grammar/syntax for conditional_get docs [ci skip]

### PR DESCRIPTION
**Pull request by gouravtiwari**

### CODE REVIEW

File: actionpack/lib/action_controller/metal/conditional_get.rb [hunk 1]
```diff
@@ -101,17 +104,19 @@ def fresh_when(object = nil, etag: object, last_modified: nil, public: false, te
...
 
-    # Sets the +etag+ and/or +last_modified+ on the response and checks it against
-    # the client request. If the request doesn't match the options provided, the
-    # request is considered stale and should be generated from scratch. Otherwise,
-    # it's fresh and we don't need to generate anything and a reply of <tt>304 Not Modified</tt> is sent.
+    # Sets the +etag+ and/or +last_modified+ on the response and checks it
+    # against the client request. If the request doesn't match the options
+    # provided, the request is considered stale and should be generated from
+    # scratch. Otherwise, it's fresh and we don't need to generate anything
+    # and a reply of <tt>304 Not Modified</tt> is sent.
```
Comment (senny) [inline actionpack/lib/action_controller/metal/conditional_get.rb]: what is the change in this paragraph? Please remove the wrapping if there is no wording change.

File: actionpack/lib/action_controller/metal/conditional_get.rb [hunk 2]
```diff
@@ -13,26 +13,29 @@ module ConditionalGet
...
     module ClassMethods
-      # Allows you to consider additional controller-wide information when generating an ETag.
-      # For example, if you serve pages tailored depending on who's logged in at the moment, you
-      # may want to add the current user id to be part of the ETag to prevent unauthorized displaying
-      # of cached pages.
+      # Allows you to consider additional controller-wide information when
+      # generating an ETag.
+      # For example, if you serve pages tailored depending on who's logged in
+      # at the moment, you may want to add the current user id to be part of
+      # the ETag to prevent unauthorized displaying of cached pages.
```
Comment (senny) [inline actionpack/lib/action_controller/metal/conditional_get.rb]: what is the change in this paragraph? Please remove the wrapping if there is no wording change.

File: actionpack/lib/action_controller/metal/conditional_get.rb [hunk 3]
```diff
@@ -13,26 +13,29 @@ module ConditionalGet
...
       #
       #   class InvoicesController < ApplicationController
       #     etag { current_user.try :id }
       #
       #     def show
-      #       # Etag will differ even for the same invoice when it's viewed by a different current_user
+      #       # Etag will differ even for the same invoice when it's viewed
+      #       # by a different current_user
```
Comment (senny) [inline actionpack/lib/action_controller/metal/conditional_get.rb]: what is the change in this paragraph? Please remove the wrapping if there is no wording change.

**Comment by senny** [comment]
From my point of view these changes don't improve much on the existing docs. They seem cosmetic. I'd rather keep the current state.

/cc @zzak

File: actionpack/lib/action_controller/metal/conditional_get.rb (hunk 1 above)
Comment (gouravtiwari) [inline actionpack/lib/action_controller/metal/conditional_get.rb]: Sure @senny, I was reading code and it was going beyond 80 lines and I had to horizontal scroll, hence made changes, but what you said also make sense. I will keep this in mind.

**Comment by gouravtiwari** [comment]
I am closing this as suggested by @senny

**Pull request closed without merging**